src/transforms/replace.jl

```diff
@@ -0,0 +1,30 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    Replace(old₁ => new₁, old₂ => new₂, ..., oldₙ => newₙ)
+
+Replaces `oldᵢ` value with `newᵢ` value in the table.
+"""
+struct Replace{K,V} <: Colwise
+  pairs::IdDict{K,V}
+end
+
+Replace() = throw(ArgumentError("Cannot create a Replace object without arguments."))
+
+Replace(pairs::Pair...) = Replace(IdDict(values(pairs)))
+
+isrevertible(::Type{<:Replace}) = true
+
+function colcache(transform::Replace, x)
+  olds = keys(transform.pairs)
+  inds = [findall(v -> v === old, x) .=> old for old in olds]
+  Dict(reduce(vcat, inds))
+end
+
+colapply(transform::Replace, x, c) =
+  map(v -> get(transform.pairs, v, v), x)
+
+colrevert(::Replace, x, c) =
+  map(i -> get(c, i, x[i]), 1:length(x))
```

Comment:
Same here, a list comprehension can improve the readability of this line.

Reply:
Made in [2f87e1d](https://github.com/JuliaML/TableTransforms.jl/pull/42/commits/2f87e1d78626a6982403f3de4ffd39eb107f4ab8)

Reply:
Which of these options do you prefer:
One line:
```julia
colrevert(::Replace, x, c) = [get(c, i, x[i]) for i in 1:length(x)]
```
Two lines:
```julia
colrevert(::Replace, x, c) = 
  [get(c, i, x[i]) for i in 1:length(x)]
```